app/notification.go

```diff
@@ -832,6 +832,9 @@ func GetExplicitMentions(message string, keywords map[string][]string) *Explicit
 				continue
 			}
 
+			// remove trailing '.', as that is the end of a sentence
+			word = strings.TrimSuffix(word, ".")
+
```

Comment:
This simply removes the period at the end of the split. So 
- `@username.` becomes `@username` and gets properly evaluated in the first section (pre-886). It used to get caught in the second block when it was split on the period.
- `@user.name.` becomes `@user.name` and gets properly caught in the first section (pre-886). In the old code it isn't caught in the first section, and it becomes `@user` and `name` after 886, thus never caught.